packages/directory/addon/routes/directory/my-directory.js

```diff
@@ -93,6 +101,9 @@ export default Route.extend({
     let user = get(this, 'user').getUser(),
         directoryParams = this.paramsFor('directory');
 
-    return this._fetchItems(user, directoryParams);
+    return this._fetchItems(user, directoryParams).then(items => ({
```

Comment:
Can we use `await` here instead of `then`? We have been following that pattern here.